### DISCUSSION TITLE
能否做一个Pake应用商店？支持上架和下载已经打包好的app，看简介，介绍，作者等，支持评论

### DISCUSSION THREAD

**Discussion by kamjin3086** [tw93/Pake#420] [Q&A]
rt
我觉得每次从大佬的release页翻应用比较费劲，后面越来越多不利于找，自己打包虽然简单但也需要费点时间，也不利于将好用的打包好的软件的分享和扩散，于是我有了这样想法，没有想太多其中对于作者和其他人的利害关系等，请指教。
另外对于初始应用很少的问题，我的思路是用爬虫或gptAPI，找出待打包的网站，用脚本批量打包应用，将打包的应用快速上架到商店上（本质是传输文件到服务器上），比如可以打包一些4399小游戏，安卓上可以装，支持触屏点击等

**Comment by momobobe** [comment]
Maybe we can instead do something like getstation.com or ferdium.org, a hub app where optimized site configuration get listed there. It could be somewhat like GUI for Pake, where you can get (or import) recipes shared by others, preview the effect and modify on the fly. I guess the packaging process should still be done locally, since if you cannot find a free service, nobody would like to afford the storage server for binaries without profit (unless only for small recipes). Instead, the packaging process could be further simplified to easy enough, just like one-click and wait, with no complicated environment configuration or whatever.

**Comment by Tlntin** [comment]
打包对方的网页上架，可能会有版权问题。比如你打包微信网页版，命名为微信，然后传到google play, 微软应用商店，最后是否会导致被告？
